exporter/datadogexporter/config_warning_deprecated.go

```diff
@@ -31,77 +31,44 @@ type renameError struct {
 	newName string
```

Comment:
What do you think about merging these files (`config_*`) into `config.go`? It looks a bit much atm. Same for `config_test.go`

<img width="313" alt="Screenshot 2022-07-06 at 15 42 36" src="https://user-images.githubusercontent.com/6686356/177553135-2f038a28-d0c2-4db4-ad4c-a3f9d0dcecca.png">